### PR TITLE
*: fix static check issues

### PR DESCRIPTION
**Pull request by Rustin-Liu**
### What problem does this PR solve? 
#13736

### Check List 

Tests 

None

Code changes

None

Side effects

None

Related changes

None

Release note

None

### CODE REVIEW

**Comment by Rustin-Liu** [comment]
PTAL~

File: executor/load_data.go [hunk 1]
```diff
@@ -241,7 +241,7 @@ func (e *LoadDataInfo) CommitWork(ctx context.Context) error {
 		case <-e.QuitCh:
 			err = errors.New("commit forced to quit")
 			logutil.Logger(ctx).Error("commit forced to quit, possible preparation failed")
-			break
+			return err
```
Comment (wjhuang2016) [inline executor/load_data.go]: It seems that the  `tasks` in line 237 is redundant, please remove it.

File: executor/explain_unit_test.go [hunk 2]
```diff
@@ -88,5 +88,5 @@ func TestExplainAnalyzeInvokeNextAndClose(t *testing.T) {
 			t.Errorf("panic test failed: without panic or close() is not called")
 		}
 	}()
-	_, err = explainExec.generateExplainInfo(tmpCtx)
```
Comment (lamxTyler) [inline executor/explain_unit_test.go]: A better fix is to assert the err?

File: executor/load_data.go (hunk 1 above)
Comment (Rustin-Liu) [reply executor/load_data.go]: It used at line 252.

File: executor/explain_unit_test.go (hunk 2 above)
Comment (Rustin-Liu) [reply executor/explain_unit_test.go]: Logged the err.

**Comment by Rustin-Liu** [comment]
PTAL~

File: executor/load_data.go (hunk 1 above)
Comment (wjhuang2016) [reply executor/load_data.go]: But it does nothing, this is an useless usage.

Comment (Rustin-Liu) [reply executor/load_data.go]: I think this val is used for counting the task count and logged on line 255.
Did you mean we do not need count this?

Comment (wjhuang2016) [reply executor/load_data.go]: Ok, my mistake.

File: executor/explain_unit_test.go [hunk 3]
```diff
@@ -71,22 +71,26 @@ func TestExplainAnalyzeInvokeNextAndClose(t *testing.T) {
 		explain:      nil,
 	}
 	// mockErrorOperator returns errors
-	mockOper := mockErrorOperator{baseExec, false, false}
-	explainExec.analyzeExec = &mockOper
+	mockOpr := mockErrorOperator{baseExec, false, false}
```
Comment (wjhuang2016) [inline executor/explain_unit_test.go]: Why rename it?

Comment (Rustin-Liu) [reply executor/explain_unit_test.go]: Because the abbreviation of the operator is Opr instead of Oper.

File: executor/explain_unit_test.go (hunk 2 above)
Comment (Rustin-Liu) [reply executor/explain_unit_test.go]: PTAL~

**Comment by Rustin-Liu** [comment]
ping~

**Comment by Rustin-Liu** [comment]
@lamxTyler  PTAL~

**Pull request merged**